### DISCUSSION TITLE
keepass does not work on certain web sites

### DISCUSSION THREAD

**Discussion by KaiGitGub** [keepassxreboot/keepassxc#6709] [Q&A] [answered]
I feel, that I will be eaten for writing this here, but yeah..

I have found out, that keepass (windows + google chrome) is not working with new cisco netacad web site:

- https://id.cisco.com/ (it is not recognizing username field - password field is working properly)

- https://connect.garmin.com/signin/ (there is a keepass logo near username, but it is not "clickable", neither login/password is working)

regards

**Comment by droidmonkey** [comment, marked as answer]
https://github.com/keepassxreboot/keepassxc-browser/issues/1358
